src/gui/editors/BBEditor.cpp

```diff
@@ -326,11 +326,15 @@ void BBTrackContainerView::clonePattern()
 
 	BBTrack *bbt = BBTrack::findBBTrack(cur_bb);
 
-	// Clone the track
-	Track *newTrack = bbt->clone();
-
-	// Track still have the TCOs which is undesirable in this case, clear the track
-	newTrack->lock();
-	newTrack->deleteTCOs();
-	newTrack->unlock();
+	if( bbt )
+	{
+		// Clone the track
+		Track *newTrack = bbt->clone();
```

Comment:
```suggestion
		BBTrack *newTrack = bbt->clone();
```
I don't have my dev machine running right now, do you mind checking if this works? It probably won't, but if it does you can remove the cast on the next line.

Reply:
Checked. Doesn't work because `clone()` always returns a `Track*`.
The static cast is fine though.

Reply:
Figures. Thanks for double checking.